### PR TITLE
adds workflow update issue

### PR DESCRIPTION
**Pull request by eastandwestwind**

### CODE REVIEW

File: .github/workflows/create_update_issue.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+name: Create Fidesdemo Update Issue
+
+on:
+  repository_dispatch:
+    types: [new-fidesops-release]
+
+jobs:
+  Create-Issue:
+    name: Create Fidesdemo Update Issue
```
Comment (PSalant726) [inline .github/workflows/create_update_issue.yml:9]:
```suggestion
    name: Create Fidesops Update Issue
```

**Review by PSalant726 — APPROVED** [review]

File: .github/workflows/create_update_issue.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+jobs:
+  Create-Issue:
+    name: Create Fidesops Update Issue
+    permissions:
+      issues: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create Issue
+        uses: imjohnbo/issue-bot@v3
```
Comment (seanpreston) [inline .github/workflows/create_update_issue.yml:15]: We ought to be careful here which bots we give permission to write to our repos. @PSalant726 / @eastandwestwind did you vet this at all?

Comment (eastandwestwind) [reply .github/workflows/create_update_issue.yml:15]: no, I didn't vet this, I copied over much of this code from fides plus

Comment (PSalant726) [reply .github/workflows/create_update_issue.yml:15]: This just wraps the official GitHub APIs, and it's referenced in the official GitHub Actions docs, here: https://docs.github.com/en/actions/managing-issues-and-pull-requests/scheduling-issue-creation

**Review by seanpreston — APPROVED** [review]

**Pull request merged**